workflow/Snakefile

```diff
@@ -129,7 +129,7 @@ rule sam_stats:
 
 rule count_reads:
     input:
-        bam=rules.sam_index.output.ibam,
+        bam=rules.map_reads.output, # use unsorted .bam files; minimap2 does not create .bam -> new input needed; also filtering may be necessary (cutadapt)
```

Comment:
Please update the comment: Just state that compressed output will not be generated. The filtering is some kind of TODO, which we ought to discuss